### PR TITLE
Add Campus Support Page

### PR DESCRIPTION
**Pull request by CannonLock**
https://osg-htc.org/web-preview/preview-cs/

### CODE REVIEW

File: campus/support.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Campus Support
+layout: text-optimized-width
+---
+
+<h1 class="mini-bar">{{ page.title }}</h1>
```
Comment (osg-cat) [inline campus/support.html:6]: Why this `mini-bar` class? It looks odd and I don’t see it on other pages. Unless there’s good reason for it, I’d like to drop the class.

**Review by osg-cat — CHANGES_REQUESTED** [review]
I made significant content changes (which was my plan all along), and otherwise left a couple questions. When these changes are available in preview mode, I’d like Christina to take a quick look, too, if she has time — I can coordinate that if you like.

File: _includes/layout/header.html [hunk 2]
```diff
@@ -34,8 +34,14 @@
...
                         </div>
                     </li>
-                    <li class="nav-item active">
-                        <a class="nav-link" href="{{ '/campus-cyberinfrastructure.html' | relative_url }}">Campuses</a>
+                    <li class="nav-item dropdown active">
+                        <a class="nav-link dropdown-toggle" href="#" id="campusNavLink" role="button" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                            Campuses
+                        </a>
+                        <div class="dropdown-menu" aria-labelledby="campusNavLink">
+                            <a class="dropdown-item" href="{{ '/campus/support.html' | relative_url }}">Support</a>
```
[the remark this answers is not in the document]
Comment (ChristinaLK) [reply _includes/layout/header.html:42]: @osg-cat I don't think there *is* an index page for the different headings on the website.

**Comment by osg-cat** [comment]
I am ok with it looking a bit empty, if the alternative is “empty” images that do not contribute to the message. The emptiness will be pressure on us to add meaningful content.
But if y’all really want pictures, go ahead!

**Comment by CannonLock** [comment]
I think images would be good as long as they add value to that page, unfortunately I cannot think of any such images. @ChristinaLK Do you have any ideas?

**Comment by CannonLock** [comment]
@osg-cat I made your changes to the header, is there any other changes you would like made before I merge?

**Comment by osg-cat** [comment]
Let’s just get it up and we can tweak as needed.

**Comment by osg-cat** [comment]
Looks great, thanks @CannonLock!

**Pull request merged**